### ISSUE TITLE
It's not obvious whether the plugin should be installed locally or remotely for each function.

### ISSUE THREAD

**Issue by ianmjones** [deliciousbrains/wp-migrate-db-pro-tweaks#19]
Sometimes the tweaks plugin should be installed locally, sometimes it should be installed on the remote, and sometimes it depends on whether it's an export, push or pull as to where the tweaks plugin should be installed.

We should make it more obvious where the tweaks plugin should be installed for each use.

**Comment by A5hleyRich** [comment]
:+1:

**Comment by JRGould** [comment]
I typically recommend installing the plugin on both sites since you could use it to fix an issue with pushing and then decide you want to pull and find that it's broken again - so best to just cover the bases. 

Other than that, I totally agree that the boilerplate would benefit from some comments that make it clear which site it needs to be active on for any scenario.

**Comment by chrislee** [comment]
Yep. Same here. I had it installed on the target site & kept erring & was about to log a case & thought to put it on the source site. Even some verbiage on the Setup instructions on the main tab would be helpful. Thank you though!
